src/wad/name.rs

```diff
@@ -55,8 +55,11 @@ impl WadName {
             Ok(WadName(name))
         }
     }
+}
 
-    pub fn from_str(value: &str) -> Result<WadName> {
+impl FromStr for WadName {
```

Comment:
This broke the tests: you can run `make test` locally to run the tests for all the crates. Could you add a `use` statement for `FromStr` in the tests module below